xcdat/dataset.py

```diff
@@ -306,10 +316,12 @@ def decode_time(dataset: xr.Dataset) -> xr.Dataset:
 
     if len(coord_keys) == 0:
         raise KeyError(
-            "Unable to map to time coordinates in this dataset to perform decoding. "
-            "Make sure that the time coordinates have the CF 'axis' or 'standard_name' "
-            "attribute set (e.g., ds['time'].attrs['axis'] = 'T' or "
-            "ds['time'].attrs['standard_name'] = 'time'), and try decoding again. "
+            "No time coordinates were found in this dataset to decode. If time "
+            "coordinates were expected to exist, make sure they are detectable by "
+            "setting the CF 'axis' or 'standard_name' attribute (e.g., "
+            "ds['time'].attrs['axis'] = 'T' or "
+            "ds['time'].attrs['standard_name'] = 'time'). Afterwards, try decoding "
+            "again with `xcdat.decode_time`."
```

Comment:
The updated `KeyError` in `decode_time()`, which is raised as a logger warning in `open_dataset()` and `open_mfdataset()`